config/config.sample.php

```diff
@@ -166,70 +170,74 @@
  */
 
 /**
- * This sets the default language on your ownCloud server, using ISO_639-1
- * language codes such as `en` for English, `de` for German, and `fr` for
- * French. It overrides automatic language detection on public pages like login
- * or shared items. User's language preferences configured under "personal ->
- * language" override this setting after they have logged in.
+ * Define the default language of your ownCloud instance
+ * Using ISO_639-1 language codes such as `en` for English, `de` for German, and `fr` for French.
+ * Overrides automatic language detection on public pages like login or shared items.
+ * User's language preferences configured under `personal -> language` override
+ * this setting after they have logged in.
  */
 'default_language' => 'en_GB',
 
 /**
- * Set the default app to open on login. Use the app names as they appear in the
- * URL after clicking them in the Apps menu, such as documents, calendar, and
- * gallery. You can use a comma-separated list of app names, so if the first
- * app is not enabled for a user then ownCloud will try the second one, and so
+ * Define the default app to open on user login
+ * Use the app names as they appear in the URL after clicking them in the Apps menu,
+ * such as files, documents or calendar ect. You can use a comma-separated list of app names,
+ * so if the first app is not enabled for a user then ownCloud will try the second one, and so
  * on. If no enabled apps are found it defaults to the Files app.
  */
 'defaultapp' => 'files',
 
 /**
- * `true` enables the Help menu item in the user menu (top right of the
- * ownCloud Web interface). `false` removes the help item.
+ * Enables or removes the Help menu item in the user menu
+ * The Help menue can be found in top right of the ownCloud Web interface.
+ * `true` enables the Help menu item, `false` removes the help item.
  */
 'knowledgebaseenabled' => true,
 
 /**
- * `true` enables avatars, or user profile photos. These appear on the User
- * page, on user's Personal pages and are used by some apps (contacts, mail,
- * etc). `false` disables them.
+ * Enables or disable  avatars or user profile photos
+ * `true` enables avatars, or user profile photos, `false` disables them.
+ * These appear on the User page, on user's Personal pages and are used by some apps
+ * (contacts, mail, etc). 
  */
 'enable_avatars' => true,
 
 /**
- * `true` allows users to change their display names (on their Personal
- * pages), and `false` prevents them from changing their display names.
+ * Allow or disallow users to change their display names
```

Comment:
```suggestion
 * Allows or disallows users to change their display names
```

What do we want to do about boolean options - `Allow or disallow` or `Allows or disallows`

@settermjd what is the "best" to do, then we be consistent?